packages/amplify-util-mock/src/utils/lambda/find-lambda-triggers.ts

```diff
@@ -2,19 +2,36 @@ import { $TSContext, pathManager, stateManager, JSONUtilities, $TSObject } from
 import * as path from 'path';
 import _ = require('lodash');
 import { ServiceName } from 'amplify-category-function';
+import { getMockSearchableTriggerDirectory } from '../mock-directory';
 
-type LambdaTriggerNames = string[];
-type LambdaTriggersMap = { [index: string] : LambdaTriggerNames};
+type LambdaTriggersMap = { [index: string] : LambdaTrigger[]};
+
+export type LambdaTrigger = {
+  name?: string,
+  config?: LambdaTriggerConfig
+}
+
+export type LambdaTriggerConfig = {
+  handler: string,
+  runtimePluginId: string,
+  runtime: string,
+  directory: string,
+  envVars: $TSObject,
+  reBuild: boolean
```

Comment:
```suggestion
  rebuild: boolean
```

Reply:
This requires a number of other references to be updated, but "rebuild" is one word.

Reply:
noted for next time.